ru.sview.sView.json

```diff
@@ -0,0 +1,141 @@
+{
+    "app-id" : "ru.sview.sView",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "21.08",
+            "add-ld-path": "."
+        }
+    },
+    "cleanup-commands": [
+        "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
+    ],
+    "command" : "sView",
+    "rename-desktop-file" : "sViewMP.desktop",
+    "rename-icon" : "sview",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* GPU acceleration */
+        "--device=dri",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Get access to the files */
+        "--filesystem=host:ro",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfs:ro"
+    ],
+    "modules" : [
+        "shared-modules/gtk2/gtk2.json",
+        {
+            "name" : "libconfig",
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url" : "https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz",
+                    "sha256": "7c3c7a9c73ff3302084386e96f903eb62ce06953bb1666235fac74363a16fad9"
+                }
+            ]
+        },
+	{
+	    "name": "wxWidgets",
+	    "rm-configure": true,
+            "cleanup": [
+                "/bin",
+                "/lib/wx",
+                "/share/bakefile"
+            ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
+		    "sha256": "96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+		},
+		{
+		    "type": "script",
+		    "dest-filename": "autogen.sh",
+		    "commands": [
+			"cp -p /usr/share/automake-*/config.{sub,guess} .",
+			"autoconf -f -B build/autoconf_prepend-include"
+		    ]
+		}
+	    ]
+	},
+        {
+            "name": "codeblocks",
```

Comment:
Why Code::Blocks is necessary here at all?

Reply:
Maybe it's a copy-paste error? But yeah, this is practically the only "blocker" for the merge.

Reply:
Because it used to be required: https://github.com/hadess/sview/blob/wip/hadess/desktop-fixes/docs/INSTALL.md#iii-codeblocks

Reply:
Code::Blocks is just an IDE. Automated scripts building setup package still relies on them on Windows platform, but Linux builds relies on plain Makefile - hence, there is no need to install Code::Blocks for building sView.